units/src/amount/unsigned.rs

```diff
@@ -15,25 +15,28 @@ use super::{
     parse_signed_to_satoshi, split_amount_and_denomination, Denomination, Display, DisplayStyle,
     OutOfRangeError, ParseAmountError, ParseError, SignedAmount,
 };
-#[cfg(feature = "alloc")]
-use crate::{FeeRate, Weight};
 
-/// An amount.
+/// An unsigned bitcoin amount.
+///
+/// The [`Amount`] type can be used to express non-negative bitcoin amounts less than [`MAX_MONEY`].
```

Comment:
I think this should just say less than 21 Million BTC instead of continue to use MAX_MONEY.  Most developers probably won't know what MAX_MONEY means really.